apps/pie-microsite/src/assets/styles/components/_list.scss

```
@@ -0,0 +1,53 @@
+@use '../fozzie-config' as f;
+
+.c-list {
+    list-style: none;
+    padding-left: 0;
+}
+
+.c-list.c-list--ordered {
+    // create a counter 'item'. Each `.c-list-item` increases the counter by 1. This value is added as the list item number in `.c-list-item::before`
+    // https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Counter_Styles/Using_CSS_counters
+    counter-reset: item;
+
+    .c-list-item {
+        counter-increment: item;
+        margin-bottom: f.$spacing-02;
+        @include f.font-size('body-l');
+        display: flex;
+
+        @include f.media('>=wide') {
+            align-items: center;
+        }
+
+        &:before {
+            content: counter(item);
+            @include f.font-size('body-s');
+            font-weight: f.$font-weight-bold;
+            margin-right: f.$spacing-03;
+            width: 25px;
+            height: 25px;
+            background: f.$color-container-inverse;
+            border-radius: 100%;
+            color: f.$color-white;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            flex: 0 0 auto;
+        }
+    }
+}
+
+.c-list.c-list--pill {
```

Same here – I'd just make this:

```suggestion
.c-list--pill {
```